.github/workflows/ci-code.yml

```diff
@@ -104,7 +104,7 @@ jobs:
         AIIDA_WARN_v3: 1
       # Python 3.12 has a performance regression when running with code coverage
       # so run code coverage only for python 3.9.
-      run: pytest -v tests -m 'not nightly' ${{ matrix.python-version == '3.9' && '--cov aiida' || '' }}
+      run: pytest --db-backend psql -m 'not nightly' tests/ ${{ matrix.python-version == '3.9' && '--cov aiida' || '' }}
```

Comment:
My predicament was that since I am anyway modifying these lines, I might as well do it here. Is it really going to be so confusing? (I guess I am just lazy to split this out, since it's kind of annoying to go through all of these files again :-D)

Reply:
Don't let @unkcpz read this, otherwise he will preach to you how we should not ignore best practices out of laziness (which I agree with, though :D)